devicesetup/src/main/res/values/themes.xml

```diff
@@ -26,7 +27,8 @@
         <item name="android:textAppearanceSmallPopupMenu">@style/PopupMenuText</item>
     </style>
 
-    <style name="PopupMenu" parent="Widget.AppCompat.Light.PopupMenu.Overflow">
+    <style name="PopupMenu" parent="Widget.AppCompat.Light.PopupMenu.Overflow"
```

Comment:
Instead of silencing this warning, why not just fix it by copying in the necessary resources?